src/api/teams/controllers/update-team.js

```diff
@@ -58,6 +63,27 @@ const updateTeamController = {
       if (!gitHubExists) {
         throw Boom.badData('Team does not exist in GitHub')
       }
+      // This is still experimental, so we're just going to log the error from this bit for now.
+      if (
+        existingTeam?.github &&
+        (await getTeamsCount(request.db, { github: existingTeam.github })) === 1
+      ) {
+        try {
+          await deleteSharedRepoAccess(request.octokit, existingTeam.github)
+        } catch (error) {
+          request.logger.error(
+            `Failed to delete ${existingTeam?.github} to the shared repos: ${error}`
+          )
+        }
```

Comment:
so have I understood this correctly? If there is 1 CDP team with the specified gitHub team then delete it and then add it back in? Feels like we could just not delete it and then log on the `204` response?